al_aws_collector.js

```diff
@@ -203,7 +203,17 @@ class AlAwsCollector {
     send(data, callback){
         var collector = this;
         var ingestType = collector._ingestType;
-        
+
+        // if the data is falsey, and empty array or an empty object, just return the callback.
+        const stringifiedData = JSON.stringify(data);
```

Comment:
The `data` should come stringified already.
I think we can also add a type check here.

Reply:
Probably worth adding function description to reflect this.